adoc/admin-migrate-caasp4.5.adoc

```diff
@@ -0,0 +1,74 @@
+== Migration to {productname} 4.5
+
+1) The OS must be connected to SCC. Otherwise the migration will not work.
+In order to reconnect your OS to SCC, you run:
+
+----
+SUSEConnect -r <your SCC key> SUSEConnect -p sle-module-containers/15.1/x86_64 -r <your SCC key>
+----
+
+2) You also need the new `zypper migration` plugin. This plugin is used to perform the needed tasks in order to migrate the node itself to the latest version of {platform}, such as updating the repositories to the new ones, and calling `zypper dup`. This plugin is provided by the `zypper-migration-plugin` package. Therefore, install the `zypper-migration-plugin` package:
+
+----
+zypper -n in zyppper-migration-plugin
+----
+
+3) Then, run the newly installed `zypper-migration` plugin (on the management node first, then on the rest of the nodes):
+
+----
+zypper migration --non-interactive --auto-agree-with-licenses
```

Comment:
no need to do it non interactive and auto agree with licenses, in the docs.

Reply:
> no need to do it non interactive and auto agree with licenses, in the docs.

Actually that is true. Changing it.